src/OpenTelemetry/Logs/LoggerProviderSdk.cs

```diff
@@ -201,7 +204,12 @@ public bool ContainsBatchProcessor(BaseProcessor<LogRecord> processor)
     }
 
     /// <inheritdoc />
-    protected override bool TryCreateLogger(string? name, out Logger? logger)
+    protected override bool TryCreateLogger(
```

Comment:
This is for Fixing:
https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/5009473252/jobs/8978448877